external-api/src/v1/controllers/estore/eStore.controller.ts

```diff
@@ -1,48 +1,33 @@
-/* eslint-disable @typescript-eslint/unbound-method */
-/* eslint-disable @typescript-eslint/no-unsafe-call */
-/* eslint-disable @typescript-eslint/no-unsafe-member-access */
-/* eslint-disable @typescript-eslint/no-unsafe-assignment */
-/* eslint-disable @typescript-eslint/no-unsafe-argument */
-/* eslint-disable @typescript-eslint/no-explicit-any */
-/* eslint-disable @typescript-eslint/no-unsafe-return */
-/* eslint-disable no-param-reassign */
-/* eslint-disable @typescript-eslint/no-misused-promises */
-/* eslint-disable @typescript-eslint/no-floating-promises */
-/* eslint-disable import/no-extraneous-dependencies */
-
-import dotenv from 'dotenv';
 import { HttpStatusCode } from 'axios';
 import { ObjectId } from 'mongodb';
 import { Request, Response } from 'express';
-import { CronJob } from 'cron';
 import { getCollection } from '../../../database';
 import { Estore, SiteExistsResponse, EstoreErrorResponse } from '../../../interfaces';
 import { ESTORE_SITE_STATUS, ESTORE_CRON_STATUS } from '../../../constants';
 import { areValidUkefIds, objectIsEmpty } from '../../../helpers';
-import { eStoreTermStoreAndBuyerFolder, eStoreSiteCreationCron } from '../../../cron';
+import { eStoreTermStoreCreationJob, eStoreSiteCreationCron } from '../../../cron';
 import { createExporterSite, siteExists } from './eStoreApi';
 import { getNowAsEpoch } from '../../../helpers/date';
 
-dotenv.config();
-
-const { ESTORE_CRON_MANAGER_SCHEDULE, TZ } = process.env;
-
 export const create = async (req: Request, res: Response) => {
   try {
+    const { body } = req as {
```

Comment:
It might be nicer to use something like we do here https://github.com/UK-Export-Finance/dtfs2/blob/main/dtfs-central-api/src/v1/controllers/utilisation-report-service/post-payment.controller/index.ts to type the request coming in as an express Request but with the body as the shape you're after. Could still do the objectIsEmpty check although if we have the time doing some kind of zod validation on the whole body would be the nicest way forward

Reply:
Really good shout, have refactored and will ensure to keep using `CustomExpressRequest`.
Have moved it to `libs/common` and will raise another PR to refactor existing `dtfs-central-api` usage.

Reply:
Nice! Do we still need these 3 lines that are casting body to an object? Should have a type now and able to run objectIsEmpty on it